### PR TITLE
Upgrade mParticle kit to Radar SDK v3

### PR DESCRIPTION
**Pull request by brettguenther**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -41,8 +41,6 @@ android {
 }
 
 dependencies {
-    api ('io.radar:sdk:2.1.9') {
-        exclude group: 'com.android.support', module: 'support-v4'
-    }
-    api 'com.android.support:appcompat-v7:28.0.0'
+    api 'io.radar:sdk:3.0.0'
+    api 'androidx.appcompat:appcompat:1.0.2'
```
Comment (nickpatrick) [inline build.gradle]: We might not need to include this if `io.radar:sdk:3.0.0` pulls it in

File: src/main/java/com/mparticle/kits/RadarKit.java [hunk 2]
```diff
@@ -49,12 +44,15 @@ private void tryTrackOnce() {
         String publishableKey = settings.get(KEY_PUBLISHABLE_KEY);
         mRunAutomatically = settings.containsKey(KEY_RUN_AUTOMATICALLY) && Boolean.parseBoolean(settings.get(KEY_RUN_AUTOMATICALLY));
 
-        Radar.initialize(publishableKey);
+        Radar.initialize(context, publishableKey);
+        Radar.setAdIdEnabled(true);
```
Comment (nickpatrick) [inline src/main/java/com/mparticle/kits/RadarKit.java:48]: Do we need to call this on iOS as well?

Comment (brettguenther) [reply src/main/java/com/mparticle/kits/RadarKit.java:48]: Yep - will handle that.

File: build.gradle (hunk 1 above)
Comment (brettguenther) [reply build.gradle]: Confirmed not needed.

**Review by samdozor — APPROVED** [review]

**Comment by samdozor** [comment]
@nickpatrick are you good with this one now or still reviewing?

**Comment by nickpatrick** [comment]
@brettguenther Let's add the `Radar.setAdIdEnabled(true)` call to the Android kit as well. Edit: Looks like it's done. Good to go.

@samdozor Thanks for your help!

**Review by nickpatrick — APPROVED** [review]

**Pull request merged**